spring-cloud-azure-appconfiguration-config/src/main/java/com/microsoft/azure/spring/cloud/config/AzureConfigBootstrapConfiguration.java

```diff
@@ -109,8 +110,14 @@ public ConfigServiceOperations azureConfigOperations(ConfigHttpClient client, Co
 
     @Bean
     public AzureConfigPropertySourceLocator sourceLocator(ConfigServiceOperations operations,
-                                                          AzureCloudConfigProperties properties) {
-        return new AzureConfigPropertySourceLocator(operations, properties);
+            AzureCloudConfigProperties properties, PropertyCache propertyCache) {
+        return new AzureConfigPropertySourceLocator(operations, properties, propertyCache);
+    }
+    
+    @Bean
+    @ConditionalOnMissingBean
+    public PropertyCache getPropertyCache() {
```

Comment:
nit: why we need to define the `PropertyCache` bean twice? Another one is in `AzureCloudConfigAutoConfiguration.java `.

Reply:
Removed duplicate in `AzureCloudConfigAutoConfiguration.java`